[4.1] fix upload multiple column

pxpm opened the pull request:
refs: #3198 

Our wrapper was preventing the column from getting the correct value.

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/Laravel-Backpack/CRUD/inspections/aecd7f73-d635-4016-983e-511447894142) was created.

tabacitu commented on src/resources/views/crud/columns/upload_multiple.blade.php:
```diff
@@ -10,7 +10,7 @@
         @foreach ($value as $file_path)
         @php
-            $column['wrapper']['href'] = $column['wrapper']['href'] ?? ( isset($column['disk'])?asset(\Storage::disk($column['disk'])->url($file_path)):asset($column['prefix'].$file_path) );
+            $column['wrapper']['href'] = ( isset($column['disk'])?asset(\Storage::disk($column['disk'])->url($file_path)):asset($column['prefix'].$file_path) );
```
Hmm I understand the problem, but wouldn't this prevent the `href` feature from ever being overwritten for the `upload_multiple` column type?

Maybe we can find a way to fix the bug, but keep the user ability to overwrite the `href`. What if we define the `href` above, in the `@php` block, if it wasn't already set, as a closure? I don't know...

pxpm replied:
I agree with you.

Or you mean something: `$column['wrapper']['href'] = $column['wrapper']['href'] ?? function($disk, $prefix, $file_path) { return ( isset($column['disk'])?asset(\Storage::disk($column['disk'])->url($file_path)):asset($column['prefix'].$file_path) ); }`

And in this line we do `$link = is_callable($column['wrapper']['href']) ? $column['wrapper']['href']($disk, $prefix, $file_path) : $column['wrapper']['href'];` ?

Let me know what you think so I can update the PR accordingly.

Best,
Pedro

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/Laravel-Backpack/CRUD/inspections/aecd7f73-d635-4016-983e-511447894142) completed: **No new issues**

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/Laravel-Backpack/CRUD/inspections/b058f11a-f714-4de2-abe4-303fab3ac8da) was created.

pxpm commented:
Just pushed a fix that addresses your concerns. 

Hope it's fine now, from my tests it works, it will only break if developer use a wrapper that is callable and don't use the same parameters that we need, so we need to add to the docs that the wrapper for this column, if they want it to be a closure, must accepts those 3 parameters. 

Best,
Pedro

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/Laravel-Backpack/CRUD/inspections/b058f11a-f714-4de2-abe4-303fab3ac8da) completed: **No new issues**

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/Laravel-Backpack/CRUD/inspections/b1f8a7f7-6ac9-4c36-be52-63ce27d91829) was created.

tabacitu commented:
Merged into `master`. It'll be available later today as a patch update, so you can get it using `composer update`.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/Laravel-Backpack/CRUD/inspections/b1f8a7f7-6ac9-4c36-be52-63ce27d91829) completed: **No new issues**
